src/main/java/sora/Tasks/Deadline.java

```diff
@@ -0,0 +1,55 @@
+package sora.Tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+/**
+ * Deadline is a Task with a deadline.
+ */
+public class Deadline extends Task {
+    /** Deadline of the deadline in String Format */
+    protected String by;
```

Comment:
Perhaps this could be better named? 'by' seems a bit ambigious on what it stands for.